integration/spark-common/src/main/scala/org/apache/spark/util/CarbonReflectionUtils.scala

```diff
@@ -65,7 +66,7 @@ object CarbonReflectionUtils {
         className,
         tableIdentifier,
         tableAlias)._1.asInstanceOf[UnresolvedRelation]
-    } else if (SPARK_VERSION.startsWith("2.2")) {
+    } else if (SPARK_VERSION.startsWith("2.2") || SPARK_VERSION.startsWith("2.3")) {
```

Comment:
Add a function SPARK_VERSION.above(2.2), so that we need not change code every place for every new version

Reply:
Fixed,added the Utility method for spark version comparison in SparkUtil.scala